src/app/auth/store/auth.actions.ts

```diff
@@ -0,0 +1,13 @@
+import { createAction, props } from '@ngrx/store';
+import { SignUpData, SignInData } from '../shared/authData.model';
+
+export const login = createAction('[AUTH] Login', props<{ loginData: SignInData }>());
+export const register = createAction('[AUTH] Register', props<{ signUpData: SignUpData }>());
+export const ifHasToken = createAction('[AUTH] Auth If Has Token');
```

Comment:
`ifHasToken`  is not very good name for actions, it's not clear what this action is representing.
Looking on redux action naming convention, actions should have names that:

1. represent _**action**_ that happened (e.g.: COUNT_INCREASED) - _more preferred_
or
1. represent **_instruction_** to do some action  (e.g.: INCREASE_COUNT)  - _less preferred_

Reply:
For this particular situation you can name it something like `checkAuthentication` or whatever